sources/packages/backend/libs/sims-db/src/entities/assessment-trigger.type.ts

```diff
@@ -36,4 +36,10 @@ export enum AssessmentTriggerType {
    * changes related to dependents.
    */
   StudentAppeal = "Student appeal",
+  /**
+   * Triggered when the Institution creates an offering change request to the
```

Comment:
`Triggered when the Institution creates an offering change request for the student which required student and SABC approval that will result in the assessment ...`?